lib/helpers/rule-helper.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Base for all rules
+ */
+'use strict';
+
+function RuleHelper(context) {
```

Comment:
Our projects use the `class.extend` module for creating classes. Please add that as a dependency and use that model.